apps/super-app/src/common/components/navbar/NavBar.css

```diff
@@ -0,0 +1,44 @@
+@import url("https://fonts.googleapis.com/css?family=Hind");
+a:visited {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 32px 10px;
+  gap: 56px;
+  position: absolute;
+  width: 360px;
+  height: 72px;
+  left: 16px;
+  top: 104px;
+  background-color: #f3fafc;
+}
+
+.navbar-icon {
+  display: inline-block;
+  color: #bec7ca;
+  text-decoration: none;
+  text-align: center;
+  align-items: center;
+  line-height: 1rem;
+}
+
+.navbar-icon-text {
+  display: block;
+  font-size: 16px;
+  font-weight: bold;
+  font-family: "hind";
+}
+
+.navbar-icon-active {
```

Comment:
This ends up repeating most of the styles from `.navbar-icon`, which isn't really what we want.
Instead, we can simplify this by thinking of classes as a "is-a" relationship, e.g. an active navbar icon is both active and a navbar icon, so we can make it two classes, like:


```
.navbar-icon {
  display: inline-block;
  color: #bec7ca;
  text-decoration: none;
  text-align: center;
  align-items: center;
  line-height: 1rem;
}

.navbar-icon.navbar-icon-active {
  color: #297b94;
}
```

The higher specificity will ensure that the `-active` variant always takes precedence.